src/docs/MLOS/MLOSUserGuide.adoc

```diff
@@ -3428,6 +3430,34 @@ NOTE: Pytorch were used to build the model architecture based on https://pytorch
 
 *How to use this task:* It should be used after the <<Predict_Image_Object_Detection_Model>> or <<Download_Model>> Task.
 
+===== Model_Explainability
+
+*Task Overview:* Explain a Deep learning Model using GradientExplainer.
+
+.Model_Explainability Task variables
+[cols="2,5,2"]
+|===
+| *Variable name* | *Description* | *Type*
+| `GPU_NODES_ONLY`
+| If True, the tasks will be executed on GPU nodes.
+| Boolean (default=True)
+| `DOCKER_ENABLED`
+| If True,  the tasks will be executed on a Docker container. link:https://hub.docker.com/r/activeeon/dlm3/[activeeon/dlm3] image is used by default. If False, the required libraries should be installed in the different nodes that will be used.
+| Boolean (default=True)
+| `IMG_SAMPLES`
+| Number of samples on which to explain the model's output.
+| Integer (default=2)
+| `IMG_LIST`
+| Choice two images to explain.
+| Integer (default=1)
+| `FEATURE_LAYER`
+| Choice the layer to explain
+| String (default=features[7] for a VGG16 model). You can use <model.features[0]> for AlexNet, <model.layer1[0].conv1> for ResNet, etc.
+|===
+
+*How to use this task:* It should be used after the <<Train_Image_Classification_Model>> Task.
```

Comment:
Maybe changing `How to use this task:` to `Usage`. (Maybe?)
<<Train_Image_Classification_Model>> Task. => task <<Train_Image_Classification_Model>>.